src/Main.java

```diff
@@ -1,5 +1,98 @@
+import java.util.Arrays;
+
 public class Main {
     public static void main(String[] args) {
+        // Задача 1
+        System.out.println(" Задача 1");
+        int[] weights = new int[]{1, 2, 3};
```

Comment:
массив надо было создать вот так
```suggestion
        int[] weights = new int[3];
```
и затем заполнить его элементы, потому что текущий способ аналогичен способами ниже
```suggestion
        int[] weights = {1, 2, 3};
```